base/common/python/pki/nssdb.py

```diff
@@ -2274,6 +2274,10 @@ def import_pkcs7(
                 n = n + 1
 
             logger.debug('Number of certs in PKCS #7: %s', n)
+            # Return if there aren't and certificates to add.
```

Comment:
Thanks for the patch! It looks good, but did you mean "any certificates" here?

Reply:
Thank you for catching the typo.  I push the fix.